### ISSUE TITLE
std::futures support instead of futures crate

### ISSUE THREAD

**Issue by xacrimon** [actix/actix#196]
As said in the title, it would be nice if say `addr.send()` could return a future from std instead of the futures crate. This would make working with actix in nightly much nicer.

**Comment by DoumanAsh** [comment]
Will be done only after proper stabilization of ecosystem.
Duplicate of https://github.com/actix/actix/issues/69
